term.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build darwin dragonfly freebsd linux,!appengine netbsd openbsd solaris
+// +build aix darwin dragonfly freebsd linux,!appengine netbsd openbsd solaris
```

Comment:
Would it be possible to add `os400` here as well, as well as in the other files? I'm looking at making IBMi binaries based on the toolchain from https://github.com/kadler/go/tree/pase-go1.15.3